temporalcli/commands.activity.go

```diff
@@ -189,3 +189,62 @@ func (c *TemporalActivityUpdateOptionsCommand) run(cctx *CommandContext, args []
 
 	return nil
 }
+
+func (c *TemporalActivityPauseCommand) run(cctx *CommandContext, args []string) error {
+	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	_, err = cl.WorkflowService().PauseActivityById(cctx, &workflowservice.PauseActivityByIdRequest{
```

Comment:
Since you are using direct gRPC instead of a high-level Go SDK API, you should set the request ID on calls yourself (same for update options from previous merge)

Reply:
add explicit "requestId" parameter.

That said - to be clear multiple "pause" calls will have the same effect, with same or different request ids.